pkg/oc/admin/prune/imageprune/prune.go

```diff
@@ -662,6 +668,25 @@ func (p *pruner) addBuildsToGraph(builds *buildapi.BuildList) []error {
 	return errs
 }
 
+// resolveISTagName parses  and tries to find it in the graph. If the parsing fails,
+// an error is returned. If the istag cannot be found, nil is returned.
+func (p *pruner) resolveISTagName(g genericgraph.Graph, referrer *kapi.ObjectReference, istagName string) (*imagegraph.ImageStreamTagNode, error) {
+	name, tag, err := imageapi.ParseImageStreamTagName(istagName)
+	if err != nil {
+		if p.ignoreInvalidRefs {
+			glog.V(4).Infof("Failed to parse ImageStreamTag name %q: %v", istagName, err)
```

Comment:
I would make this `glog.Warningf` (guess users should see what they are ignoring).

Reply:
@mfojtik done